content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+---
+
+![Gateway API Logo](gateway-api-logo.png)
+**Authors:** 
+GRPCRoute - Richard Belleville (Google), 
+ParentRef Port - Frank Budinsky (IBM),
+Gateway Client Certificate Validation - Arko Dasgupta (Tetrate),
+GAMMA - John Howard (Solo.io),
+Conformance Profiles and Reports - Mattia Lavacca (Kong), Christine Kim (Isovalent),
+Session Persistence and BackendLBPolicy - Grant Spence (Red Hat), Gina Yeh (Google),
+TLS Terminology Clarifications - Candace Holman (Red Hat),
+and other reviewer and release note contributors.
```

Comment:
```suggestion

```

Reply:
IIUC these were the authors who wrote the copy for each section (which is distinct but may have some overlpa with the contributors who helped design and implement the respective functionality).